cassandra/cqlengine/models.py

```diff
@@ -1021,8 +1022,8 @@ def _transform_column(col_name, col_obj):
         for col in column_dict.values():
             columns.resolve_udts(col, udts)
 
-        for user_type in set(udts):
-            user_type.register_for_keyspace(klass._get_keyspace())
+        for user_type, conn in product(set(udts), connection._connections.values()):
+            user_type.register_for_keyspace(klass._get_keyspace(), connection=conn)
```

Comment:
This would probably work. However, It might be better to keep the 'connection' logic **only** in the management commands. I think It  will become confusing if we start adding some of that logic in different places. What do you think?